extensions/amp-story-interactive/0.1/amp-story-interactive-slider.js

```diff
@@ -44,11 +43,18 @@ const buildSliderTemplate = (element) => {
             value="0"
           />
           <div class="i-amphtml-story-interactive-slider-bubble"></div>
+          <div
+            class="i-amphtml-story-interactive-slider-average-indicator"
+          ></div>
```

Comment:
We can make this match the Figma more closely by using three span elements and transitioning them.

https://user-images.githubusercontent.com/3860311/128390970-877883a9-72b3-4128-9187-4c5216bda8d0.mp4



Suggested changes commit:
https://github.com/ampproject/amphtml/commit/07a2a9301437caacc5ac069c71b12086a2cd00ec